dataframe/aggregation.go

```diff
@@ -4,30 +4,46 @@ import "fmt"
 
 // Aggregation defines how to aggregate rows of a group of rows in a Groups
 // instance.
+//
+// Deprecated: This package is deprecated and will be removed in a future.
```

Comment:
```suggestion
// Deprecated: This package is deprecated and will be removed in the next major release.
```

Two opinions here:
- I think being clear about when it is going out of commission is better.
- If you don't like that for some reason, I would change `a future` for `the future`. You can check me with a native speaker but I think `the` is more appropriate here.


I'm making this comment just once to avoid polluting the PR with the same comment over and over again.

Reply:
made the change to use "next major release"